### PR TITLE
fix: update the list of dashboards after deleting one

### LINKED ISSUE #540: [ui] The list of dashboards is not updated after a dashboard is deleted
### Which ZincObserve functionalities are the source of the bug?

dashboards

### Is this a regression?

Yes

### Steps to reproduce

- Go to ‘Dashboards’ page.
- Create a dashboard if there is none.
- Delete any dashboard.

### Expected result

The dashboard should be removed from the list.

### Actual result

The list of dashboards is not updated. Reloading the page updates the list.

### Please provide a link to a minimal reproduction of the bug

_No response_

### Please provide the exception or error you saw

_No response_

### Please provide the version you discovered this bug in (check about page for version information)

```true
v0.3.2-51-gf50bf4e
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by vvv**
Closes #540

### CODE REVIEW

File: web/src/views/Dashboards/Dashboards.vue [hunk 1]
```diff
@@ -276,19 +276,17 @@ export default defineComponent({
...
 
-    const removeDashboard = async () => {
+    const deleteDashboard = async () => {
       if (selectedDelete.value) {
         const dashboardId = selectedDelete.value.id;
         await dashboardService
           .delete(store.state.selectedOrganization.identifier, dashboardId)
           .then((res) => {
-            const dashboardList = JSON.parse(
-              JSON.stringify(toRaw(store.state.allDashboardList))
```
Comment (vvv) [inline web/src/views/Dashboards/Dashboards.vue:286]: We need an object. We have it. Case closed.

`JSON.parse(JSON.stringify(obj))` pattern feels generally wrong to me. I'm not a JavaScript expert though, so maybe it is useful, [if controversial](https://medium.com/@pmzubar/why-json-parse-json-stringify-is-a-bad-practice-to-clone-an-object-in-javascript-b28ac5e36521), in some cases.

File: web/src/views/Dashboards/Dashboards.vue [hunk 2]
```diff
@@ -276,19 +276,17 @@ export default defineComponent({
       });
     });
 
-    const removeDashboard = async () => {
+    const deleteDashboard = async () => {
```
Comment (vvv) [inline web/src/views/Dashboards/Dashboards.vue:279]: Rust code, HTML labels, and (soon) the Open API specification use verb ‘delete’. There is no reason to deviate here.

**Review by hengfeiyang — APPROVED** [review]

**Comment by ktx-kirtan** [comment]
@hengfeiyang We need to hold this for now. As the changes are already done in https://github.com/zinclabs/zincobserve/pull/514#discussion_r1163588711

**Comment by vvv** [comment]
@ktx-kirtan Shall we close this PR then?

**Comment by ktx-kirtan** [comment]
@vvv Yes, we can close this.

**Comment by vvv** [comment]
@ktx-kirtan Can we merge this PR please? Dashboard deletion UI is broken, and the fix is simple.

**Comment by ktx-kirtan** [comment]
Yes, it is because #514  is pending to merge. But we can merge this as well.

**Pull request merged**